<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditThrone</class>
 <widget class="QDialog" name="AddEditThrone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Throne</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter an Alias (friendly name) for your Throne and its address. The address should be in the format 123.456.789.123:9340 or akjdsafxjkhasdf.onion:9340. You must send exactly 10 000 CRW to the collateral address.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Alias*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="aliasLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Address*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="AddEditAddressPasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../dash.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>PrivKey*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="privkeyLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="AddEditPrivkeyPasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../dash.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TxHash*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txhashLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="AddEditTxhashPasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../dash.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Output Index*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputindexLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../dash.qrc"/>
 </resources>
 <connections/>
</ui>
